src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ spring:
     datasource:
         url: jdbc:mysql://localhost:3306/swim?autoReconnect=true
         username: root
-        password: root
+        password: 1234
```

Comment:
this will break other peoples build.  Assure your mysql pwd is root, or keep this change localy just for yourself.